internal_jtimon.go

```diff
@@ -3,39 +3,25 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	gnmi "github.com/Juniper/jtimon/gnmi/gnmi"
+	na_pb "github.com/Juniper/jtimon/telemetry"
 	"log"
 	"os"
 	"regexp"
 	"strings"
-
-	gnmi "github.com/Juniper/jtimon/gnmi/gnmi"
-	na_pb "github.com/Juniper/jtimon/telemetry"
 )
 
 // InternalJtimonConfig type
 type InternalJtimonConfig struct {
 	DataLog       string `json:"data-log-file"`
+	CsvLog        string `json:"csv-log-file"`
+	CsvStats      bool   `json:"csv-stats"`
```

Comment:
why is this knob needed here? if csv-log-file is supplied, we can assume we need to dump in csv rite ? Else no csv dump.

Reply:
removed "CsvStats". Added checks based on CsvLog